cirq-google/cirq_google/engine/engine.py

```diff
@@ -126,6 +126,28 @@ def copy(self) -> 'EngineContext':
     def _value_equality_values_(self):
         return self.proto_version, self.client
 
+    def serialize_program(
+        self, program: cirq.AbstractCircuit, serializer: Optional[Serializer] = None
+    ) -> any_pb2.Any:
+        if not isinstance(program, cirq.AbstractCircuit):
+            raise TypeError(f'Unrecognized program type: {type(program)}')
+        if serializer is None:
+            serializer = self.serializer
+        if self.proto_version != ProtoVersion.V2:
+            raise ValueError(f'invalid program proto version: {self.context.proto_version}')
+        program = serializer.serialize(program)
+        return _pack_any(program)
+
+    def serialize_run_context(self, sweeps: List[cirq.Sweep], repetitions: int) -> any_pb2.Any:
```

Comment:
Since these are now public, can you add unit tests to cirq-google/cirq_google/engine/engine_test.py?